### ISSUE TITLE
Wrong parser

### ISSUE THREAD

**Issue by Hack-y** [Kloct/tera-client#8]
Hello @Kloct, i tried to update my clientless but the parser (or something else) broked my scripts.

When i try to send `C_REQUEST_CONTRACT`, this isn't parsed correctly and i never receive `S_REQUEST_CONTRACT` back.
(With clientless non updated, this work fine, i have my `S_REQUEST_CONTRACT` back).

I already check if my def was good and it's.

I think the buffer lenght is wrong but i'm not sure.

If you or Pinkie can check it ;) i didn't found solution...

If you try for exemple to send trade request with :
```
mod.send('C_REQUEST_CONTRACT', 1, {
	type: 3,
	unk2: 0,
	unk3: 0,
	unk4: 0,
	name: "NikeName"
});
```
This doesn't work because you never receive `S_REQUEST_CONTRACT` back ...

Thank you for your work and for your help ;)

**Comment by michaelcarno** [comment]
Hi o/. I'm not Kloct, but Try this: 

`mod.send("C_REQUEST_CONTRACT",1,{
             type:  3
             data: Buffer.from([0])
        })
`

**Comment by Kloct** [comment]
There is an issue with the new parser upstream with undefined bytes fields.
I told Pinkie about it a few weeks ago and he said it's more of an issue with the vanilla client/server but it only became a problem with the new parser.
@michaelcarno's workaround should work fine to solve this 👍

**Comment by Kloct** [comment]
Just merged a fix for this from proxy 6be43f7
Closing this issue now
